src/python_testing/TC_ACL_2_8.py

```diff
@@ -270,27 +299,50 @@ async def test_TC_ACL_2_8(self):
             fabricFiltered=True
         )
 
-        # Below event filtering and parsing is currently required in the event that the DUT is not reset before running this test.
-        added_event, changed_event = self._get_relevant_acl_events(events, self.th1.nodeId, [self.th1.nodeId, 1111])
-        logging.info(f"TH1 Events: added_event={added_event}, changed_event={changed_event}")
+        if force_legacy_encoding:
```

Comment:
Same comment here as on ACL-2.6 - need to confirm whether this is required. Otherwise the code looks reasonable.